pyproject.toml

```diff
@@ -72,33 +82,33 @@ select = [
   "UP",     # pyupgrade
   "W",      # pycodestyle
   "YTT",    # flake8-2020
-  # "A",    # flake8-builtins
-  # "ANN",  # flake8-annotations
-  # "ARG",  # flake8-unused-arguments
-  # "B",    # flake8-bugbear
   # "COM",  # flake8-commas
   # "CPY",  # flake8-copyright
   # "D",    # pydocstyle
   # "DOC",  # pydoclint
   # "EM",   # flake8-errmsg
-  # "ERA",  # eradicate
-  # "FA",   # flake8-future-annotations
   # "FAST", # FastAPI
   # "FBT",  # flake8-boolean-trap
-  # "FIX",  # flake8-fixme
-  # "N",    # pep8-naming
-  # "PIE",  # flake8-pie
-  # "PL",   # Pylint
   # "T20",  # flake8-print
   # "TD",   # flake8-todos
   # "TRY",  # tryceratops
 ]
 ignore = [
+  "ANN101",  # missing type annotation for `self`
+  "B904",    # raise-without-from-inside-except
+  "FIX002",  # line-contains-todo
   "ISC001",  # may conflict with ruff format
+  "PLW2901", # `for` loop variable overwritten
 ]
 
 [tool.ruff.lint.mccabe]
 max-complexity = 12
 
+[tool.ruff.lint.pylint]
+allow-magic-value-types = ["bytes", "int", "str"]
+max-args = 8  # Default: 5
+
 [tool.ruff.lint.per-file-ignores]
+"nntp/nntp.py" = ["A002", "ARG002"]
+"nntp/utils.py" = ["N806"]
```

Comment:
How many instances of each of these are there? Is it worth fixing these instead? I'm happy to do this in a follow-up.

Reply:
Thanks! I'll fix the ones that make sense in a follow up.